torch/export/exported_program.py

```diff
@@ -289,6 +269,17 @@ def buffers(self) -> Collection[str]:
             if isinstance(s.target, str)
         ]
 
+    # A list of lifted constant tensors
+    @property
+    def lifted_tensor_constants(self) -> Collection[str]:
+        # TODO Make this tuple.
```

Comment:
We can make this tuple now?

Reply:
Idk, I just copied from parameters/buffers above.